web/src/reducers/budget.js

```diff
@@ -239,7 +239,7 @@ const buildBudget = bigState => {
 
     activity.statePersonnel.forEach(person => {
       Object.entries(person.years).forEach(([year, { amt, perc }]) => {
-        const cost = n(amt) * n(perc) / 100;
+        const cost = n(amt) * n(perc);
```

Comment:
@brendansudol Just out of curiosity, how long did it to take to find this line to change?

Reply:
3 seconds

Reply:
by the way, i'm also totally fine keeping this number as integer based, instead of float (and then doing the conversion to/from in the react component for display purposes